config/redirects

```diff
@@ -46,5 +46,9 @@ raw: ${prefix}/stable -> ${base}/current/
 (v5.2-master]: ${prefix}/${version}/validate-signatures/ -> ${base}/${version}/
 (v5.2-master]: ${prefix}/${version}/builders/ -> ${base}/${version}/
 (v5.2-master]: ${prefix}/${version}/bson/ -> ${base}/${version}/
-(v5.2-master]: ${prefix}/${version}/installation/ ->
-${base}/${version}/get-started/download-and-install/
+(v5.2-master]: ${prefix}/${version}/installation/ -> ${base}/${version}/get-started/download-and-install/
```

Comment:
Q: Would putting an inclusive range instead here, [v5.3-master], do the same thing?

Reply:
I think so: https://www.mongodb.com/docs/meta/redirects/#rules